### ISSUE TITLE
Mouseless tray icons interaction

### ISSUE THREAD

**Issue by i3bot** [i3/i3#1048]
[Originally reported by anrock623@…]
(Well, title says it all. 

i3wm, as a tiled windows manager, supposed to be effectively controlled by keyboard only. That's mostly true. But there is, afaik, no way to interact with tray icons\indicators by means of keystrokes. And that, imo, should be fixed.

I don't know exactly how to achieve that goal, maybe special tray-mode which somehow allows user to simulate lmb and rmb on tray icons.
Like [meta]+[i], [shift]+1 to lmb on leftmost icon in tray and [ctrl]+1 for rmb. A little bit clumsy, but you get the idea.
